packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5706,13 +5730,17 @@ export default class Meeting extends StatelessWebexPlugin {
               remoteQualityLevel: this.mediaProperties.remoteQualityLevel,
             },
           })
-          .then(() => {
-            if (startShare) {
-              return this.requestScreenShareFloor();
-            }
+          .then(() =>
+            this.enqueueMediaUpdate(MEDIA_UPDATE_TYPE.LAMBDA, {
+              lambda: async () => {
+                if (startShare) {
+                  return this.requestScreenShareFloor();
+                }
 
-            return Promise.resolve();
-          })
+                return undefined;
```

Comment:
should this return promise.reject? if we return undefined maybe we should write a log? especially since this is kind of hacky non-roap protocol

Reply:
if startShare is not true, it just means that we're not starting a share, because maybe we're just stopping a share or just changing the share stream, so it's not an error condition, it's just that we have nothing to do, and typescript was complaining when I had no `return` or an empty `return` so I'm returning undefined